vertical-pod-autoscaler/pkg/recommender/input/controller_fetcher/controller_fetcher_test.go

```diff
@@ -117,6 +119,70 @@ func TestControllerFetcher(t *testing.T) {
 				Name: "test-deployment", Kind: "Deployment", Namespace: "test-namesapce"}}, // Deployment has no parent
 			expectedError: nil,
 		},
+		{
+			key: &ControllerKeyWithAPIVersion{ControllerKey: ControllerKey{
+				Name: "test-statefulset", Kind: "StatefulSet", Namespace: "test-namesapce"}},
+			objects: []runtime.Object{&appsv1.StatefulSet{
+				TypeMeta: metav1.TypeMeta{
+					Kind: "StatefulSet",
+				},
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      "test-statefulset",
+					Namespace: "test-namesapce",
+				},
+			}},
+			expectedKey: &ControllerKeyWithAPIVersion{ControllerKey: ControllerKey{
+				Name: "test-statefulset", Kind: "StatefulSet", Namespace: "test-namesapce"}}, // Deployment has no parrent
```

Comment:
s/Deployment/Statefulset/
s/parrent/parent

Can you also update the other comments?

Reply:
okay